app/case/server.py

```diff
@@ -64,3 +64,16 @@ def update_status(deed_id):
             return jsonify(case_status=case_status), status.HTTP_200_OK
         else:
             abort(status.HTTP_400_BAD_REQUEST)
+
+    @blueprint.route('/case/<case_id>/deed', methods=['POST'])
+    def update_case_deed(case_id):
+        case = Case.get(case_id)
+
+        if case is None:
+            abort(status.HTTP_404_NOT_FOUND)
+
+        deed_id = request.data['deed_id']
+        case.deed_id = deed_id
+
+        case.save()
+        return str(status.HTTP_200_OK)
```

Comment:
also on the confirmation, don't design API's just for humans, computers consume API's most the time. I think if you are not returning anything then a bit of JSON along the lines of when you get a status 200 you get 

```
{
    "status" : 200
}
```
or something similar